packages/KDropdownMenu/KDropdownItem.vue

```diff
@@ -127,7 +127,20 @@ li.k-dropdown-item {
   line-height: 1;
 
   &.has-divider {
-    border-top: 1px solid var(--grey-200) !important;
+    --KDropdownItemDividerContainerHeight: var(--spacing-lg);
+    --KDropdownItemDividerPosition: calc((#{var(--KDropdownItemDividerContainerHeight)} / 2 + 1px) * -1);
+    position: relative;
+    margin-top: var(--KDropdownItemDividerContainerHeight);
+
+    &:before {
+      position: relative;
+      display: block;
+      content: '';
+      height: 1px;
+      width: 100%;
+      top: var(--KDropdownItemDividerPosition);
```

Comment:
Instead of allowing an override of the values (since they're CSS variables technically someone could set them and override) can we use Sass variables instead? This keeps them in-sync but prevents an override.

```suggestion
    $k-dropdown-item-divider-container-height: var(--spacing-lg);
    $k-dropdown-item-divider-position: calc((#{var(--KDropdownItemDividerContainerHeight)} / 2 + 1px) * -1);
    position: relative;
    margin-top: $k-dropdown-item-divider-container-height;

    &:before {
      position: relative;
      display: block;
      content: '';
      height: 1px;
      width: 100%;
      top: $k-dropdown-item-divider-position;
```